pages/about/principal.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Mount Sion CBS Secondary School</title>
+    
+    <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.css">
+  <link src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.3.0/semantic.min.js">
+   <script src="//code.jquery.com/jquery-2.2.1.js"></script>
+
+</head>
+ <body>
+    
+<div class="ui container">
+    <img class="ui centered medium image" src="../../assets/images/logo.jpg">
+        
+<div class="stackable ui tabular menu">
+    <a class="item" href="../../index.html">
+        Home 
+    </a>
+        <div class="ui simple dropdown item">
+            About
+            <i class="dropdown icon"></i>
+            <div class="menu">
+                <div class="item" href="./pages/about/principal.html"> Principal's Welcome</div>
+                <div class="item" href="./pages/about/history.html"> History</div>
```

Comment:
Can you change these divs to `a` to make them clickable

Reply:
:+1: